OrbitGl/CaptureWindow.cpp

```diff
@@ -537,8 +536,8 @@ void CaptureWindow::DrawScreenSpace() {
   const Color kBackgroundColor(70, 70, 70, 255);
   float margin_x1 = getWidth();
   float margin_x0 = margin_x1 - vertical_margin;
-
-  Box box(Vec2(margin_x0, 0), Vec2(margin_x1 - margin_x0, canvasHeight - height), z);
+  Box box(Vec2(margin_x0, time_graph_.GetLayout().GetSliderWidth()),
+          Vec2(margin_x1 - margin_x0, canvasHeight - height), GlCanvas::Z_VALUE_MARGIN);
```

Comment:
Good catch! Thanks. 

nit. With the z-ordering fixed, we could simplify further with: 

Box box(Vec2(margin_x0, 0), Vec2(margin_x1 - margin_x0, canvasHeight), z);

It would eliminate the potential seam between the margin and the slider. But this is not a huge deal, feel free to disregard.